apollo-runtime/src/main/java/com/apollographql/android/cache/normalized/NormalizedCacheStore.java

```diff
@@ -0,0 +1,17 @@
+package com.apollographql.android.cache.normalized;
+
+import java.util.Collection;
+
+public abstract class NormalizedCacheStore {
```

Comment:
Are all methods safe? I mean do we expect any exceptions while we reading/merging etc?